christianR15c/kisi-node-express-web-app
# create web page using express

### PR Description

create a web page using ExpressJs

### Description of tasks that were expected to be completed

- create a web page using express
- defining basic SEO: Meta title, description, open-graph image, canonical URL and open graph URL

### How has this been tested?

- clone this repo
- `git checkout ft-schema-create` 
- run  `npm Install`
- run `npm dev`
- open localhost:3000 in your favourite browser you will be able to see a web page

### Number of Commits
- this branch has 1 commit

### Screenshots 
![home](https://user-images.githubusercontent.com/50244289/180416221-564c3daa-41ec-4048-9848-ac824b24d5d8.png)


### Please check this Checklist before you submit your PR:

- [x] My test coverage meets the set test coverage threshold
